pykern/util.py

```diff
@@ -14,6 +14,11 @@
 
 _DEFAULT_ROOT = "run"
 _DEV_ONLY_FILES = ("setup.py", "pyproject.toml")
+_STRANGE_CONTROL_CODES = [
+    bytes((n,))
+    for n in range(0, 32)
+    if bytes((n,)) not in b"\x07\x08\t\n\x0b\x0c\r\x1b"
```

Comment:
instead of a byte string, simple compare the number to a set of the numbers. You could document them:

`_VALID_ASCII_CONTROL_CODES = frozenset(0x7, 0x8,,,)`